packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -59,14 +119,14 @@ function getWorkerTests(opts: { experimental: boolean }): WorkerTestConfig[] {
 			},
 			{
 				template: "queues",
-				variants: ["TypeScript", "JavaScript"],
+				variants: ["ts", "js"],
 				// Skipped for now, since C3 does not yet support resource creation
 				verifyDeploy: null,
 				verifyPreview: null,
 			},
 			{
 				template: "scheduled",
-				variants: ["TypeScript", "JavaScript"],
+				variants: ["ts", "js"],
```

Comment:
All the variant name is changed as 
1. It is now passed as `--lang [lang]` instead of going through a prompt and have the test script figure out the right answer.
2. They are part of the project name and might become too long depending on the template name. It should be less likly to be over 58 chars now.